src/recipes.jl

```diff
@@ -654,3 +663,38 @@ function update_discretized!(disc, pathes)
         disc[i][] = discretize(p)
     end
 end
+
+"""
+    update_arrow_shift!(g, gp, edge_paths::Vector{<:AbstractPath{PT}}, shift, to_angle, to_point) where {PT}
+
+Checks `arrow_shift` attr so that `arrow_shift = 1` gets transformed so that the arrowhead for that edge
+lands on the surface of the destination node.
+"""
+function update_arrow_shift!(g, gp, edge_paths::Vector{<:AbstractPath{PT}}, shift, to_angle, to_point) where {PT}
+    arrow_shift = Vector{Float32}(undef, ne(g))
+
+    #node attr
+    node_marker = gp.node_marker[] 
```

Comment:
`node_marker`, `arrow_marker` and `arrow_size` might be vectors, so you need to lookup using `getattr` for each destination node inside the loop.

Reply:
ok, fixed